### DISCUSSION TITLE
I want to implement the inscribe and send functions with nodejs, so that I can connect to my xverse wallet or okx wallet and send. Can you help me, or have code examples?

### DISCUSSION THREAD

**Discussion by RunlaiDeng** [ordinals/ord#2219] [Q&A]
I want to implement the inscribe and send functions with nodejs, so that I can connect to my xverse wallet or okx wallet and send. Can you help me, or have code examples?

**Comment by Psifour** [comment]
This repo is a reference implementation. As such, the entire thing is a code example.

The part relevant to your request is, https://github.com/ordinals/ord/blob/master/src/subcommand/wallet/inscribe.rs

**Comment by ghost** [comment]
is there any chance to work with you?
